gateway-shell/src/main/java/org/apache/knox/gateway/shell/table/KnoxShellTable.java

```diff
@@ -122,6 +124,10 @@ public KnoxShellTable select(String cols) {
   }
 
   public KnoxShellTable sort(String colName) {
+    return sort(colName, SortOrder.ASCENDING);
+  }
+
+  public KnoxShellTable sort(String colName, SortOrder order) {
```

Comment:
Does public make sense here? Can SortOrder be passed in from the shell or will it always need to be a string?

Reply:
Thanks for the review! It does need to be public as other external applications such as KnoxLine will be using it. SortOrder would be passed in from the shell as SortOrder.DESCENDING.